docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,77 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
 - [Performance Tuning](https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html )
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+The properties mentioned in this section can be found in [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+### Auto start Alluxio masters and workers
+After installed Alluxio masters and workers, use this [script](../../scripts/alluxio/alluxio-systemd.sh) to create systemd services and start masters and workers.
```

Comment:
```suggestion
After installing Alluxio masters and workers, use this [script](../../scripts/alluxio/alluxio-systemd.sh) to create systemd services and start masters and workers.
```

Reply:
Done

Reply:
so I think I would still like to see a small section in this doc about reliability and using something like systemd to manage the processes. This way if users don't use our docker image they can set up themselves.  We don't need the full script and instructions perhaps just a general statement about using systemd services